net-im/discord/discord-0.0.58-r1.ebuild

```diff
@@ -123,7 +122,8 @@ src_install() {
 	# See #903616 and #890595
 	[[ -x chrome_crashpad_handler ]] && doins chrome_crashpad_handler
 
-	dosym "${DESTDIR}/${MY_PN^}" "/usr/bin/${MY_PN}"
+	exeinto "/usr/bin"
+	newexe "${T}/launcher.sh" "discord" || die "failing to install launcher"
```

Comment:
also what happens with the `.application` file, do we need to update that too?

Reply:
You mean the `.desktop` file? It is updated in [line 86](net-im/discord/discord-0.0.58-r2.ebuild#L86).

Reply:
ah ok thx